osidb/models.py

```diff
@@ -699,10 +717,17 @@ def _validate_embargoed_source(self):
         embargoed = self.acl_read == [
             uuid.UUID(acl) for acl in generate_acls([settings.EMBARGO_READ_GROUP])
         ]
-        if embargoed and FlawSource(self.source).is_public():
-            raise ValidationError(
-                f"Flaw is embargoed but contains public source: {self.source}"
-            )
+        if embargoed and (source := FlawSource(self.source)) and source.is_public():
+            if source.is_private():
+                self.alert(
```

Comment:
This is probably OK for now but in the future we should make the alerts actionable so people do not start to ignore them - when there is nothing one can do to suppress the alert they would just stop looking at them.

Reply:
What's your idea of actionable? Maybe the alert could instead be some sort of "info" instead of a warning

Reply:
Here the only thing I can think about is the checkmark of some sort but maybe we could later solve it by converting into a workflow step. I would just leave it for now and think about this more generally in the future.